src/pages/dashboard.js

```diff
@@ -309,24 +222,83 @@ const DashboardSideBar = ({ currentUser, allSkills }) => {
       const newOption = createOption(newValue)
```

Comment:
curious why we need this setTimeout and forcing the user to wait 1 second when it may not be needed?